agent/consul/server_test.go

```diff
@@ -1138,10 +1143,11 @@ func TestServer_CALogging(t *testing.T) {
 	// Setup dummy logger to catch output
 	var buf bytes.Buffer
 	logger := log.New(&buf, "", log.LstdFlags)
+	logger2 := testutil.Logger(t)
```

Comment:
`logger2` needs to send its logs into `buf` instead of `testing.T` here since this is a test that tests logging.

Reply:
Good catch!